### PR TITLE
Improve documentation

### LINKED ISSUE #2: Add a simple example
Must involve two nodes.

### LINKED ISSUE #14: Document how we translate between Zig and Erlang types

### PR DESCRIPTION
**Pull request by z-silver**
Closes #2, closes #14.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -0,0 +1,17 @@
+* 🌠 Zerl: Making Erlang + Zig communication EZ
```
Comment (EduardoLR10) [inline README.org]: Please remove the emoji.

File: README.org [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+#+html: <img src="images/logo.png" alt="The Zerl Logo" style="width: 25%;">
+
+/*WORK IN PROGRESS*/
```
Comment (EduardoLR10) [inline README.org]: This does not work in the preview for some reason, please remove the asteriscs.

File: doc/types.org [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+** Structs
+
+Structs are converted to maps in order to allow fields with default values,
+optional fields, and to not have to worry about field ordering on the Erlang
+side.
+
+Extern structs are currently treated exactly the same as normal structs, but
+since their fields have a guaranteed order we do want to allow tuples to map
+to them. However, it's not clear if they should be encoded as tuples when
+they are sent by themselves, outside of a tagged union, or if or how we should
```
Comment (EduardoLR10) [inline doc/types.org]: Is there a typo? `or if or`?

File: examples/README.org [hunk 4]
```diff
@@ -0,0 +1,14 @@
+* Zerl Examples
+
+In order to run these examples, you must have both Zig and Erlang installed.
```
Comment (EduardoLR10) [inline examples/README.org:3]: Shouldn't we mention the flake?

File: examples/cards/cards.zig [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+const Reply = union(enum) {
+    say: [:0]const u8,
+    halved: i31,
+    shuffled: []Card,
+    ok: void,
+    @"error": Dealer_Error,
+};
+
+const deck: [52]Card = blk: {
```
Comment (EduardoLR10) [inline examples/cards/cards.zig:59]: Add a comment describing what this magic trick does.

**Review by EduardoLR10 — CHANGES_REQUESTED** [review]

File: examples/cards/cards.zig (hunk 5 above)
Comment (z-silver) [reply examples/cards/cards.zig:59]: I object to calling it magic, but I will add a comment.

**Review by EduardoLR10 — APPROVED** [review]

**Pull request merged**